docs/content-guidelines.md

```diff
@@ -96,6 +97,33 @@ Use one of these three options for alert blocks:
 >**WARNING:** Very important information
 ```
 
+## API Documentation
+
+The [Core](luigi-core-api.md) and [Client API](luigi-client-api.md) documents are different from the rest of Luigi documentation. They are automatically generated by a script. Instead of adding documentation directly to the `.md` files in the `luigi/docs` folder, you need to:
+
+1. Add documentation as comments to the `.js` file where the API function is contained. First, write a description of what the function does. Then, include extra categories preceded by tags (if relevant):
+  * `@memberof` - defines the larger category the function is part of, e.g. `LuigiNavigation` or `Authorization`
+  * `@param` - parameters the function takes, e.g. `{boolean}`, `{string}`, `{Object}` followed by any specific name if applicable
+  * `@returns` - type and description of what the function returns, e.g. `{boolean}`, `{string}`, `{Object}` followed by any specific name if applicable
+  * `@example` - example of how the function is used
+  * `@since` - earliest Luigi version offering this feature
```

Comment:
```suggestion
  * `@example` - one or more examples of how the function is used
  * `@since NEXTRELEASE` - earliest Luigi version offering this feature
```